src/github_review_prompts/utils/github_integration.py

```
@@ -0,0 +1,812 @@
+
+        return workflow
+
+    def create_resolution_milestone(
+        self, owner: str, repo: str, pr_number: int, total_comments: int
+    ) -> Dict[str, Any]:
+        """解決追跡用のマイルストーンを作成"""
+        try:
+            milestone_data = {
+                "title": f"PR #{pr_number} 範囲外コメント対応",
+                "description": f"PR #{pr_number} の {total_comments}件の範囲外コメント対応を追跡するマイルストーン",
+                "due_on": (datetime.now() + timedelta(days=14)).isoformat(),  # 2週間後
```

Ruff (0.12.2)

Undefined name `timedelta`

(F821)